sys/linux/bpf_lsm.txt

```diff
@@ -9,7 +9,10 @@ syz_btf_id_by_name$bpf_lsm(name ptr[in, string[bpf_lsm_func_names]]) bpf_lsm_btf
 
 bpf$BPF_LSM_PROG_LOAD(cmd const[BPF_PROG_LOAD], arg ptr[in, bpf_lsm_prog], size len[arg]) fd_bpf_prog (timeout[500])
 
-type bpf_lsm_prog bpf_prog_t[const[BPF_PROG_TYPE_LSM, int32], const[BPF_LSM_MAC, int32], bpf_lsm_btf_id, const[0, int32]]
+# NEED: After #2035 is merged (which we can't for now due to pahole issues),
```

Comment:
Is this the typical way to refer to a PR in a comment, or should it be a full URL?

Reply:
So I did a `rg "\#\d"` and it shows a few
```
socket_ip_tunnel.txt
53:# Repeat IP tunnel syscall variants here until #1913 is resolved.

dev_video4linux.txt
957:#Comeback and add others from include/uapi/linux/videodev2.h#1617. Once after syz-extract able to detect with special chars

bpf_lsm.txt
12:# NEED: After #2035 is merged (which we can't for now due to pahole issues),

init.go
258:            // and/or if we mount tmpfs for sandbox=none (#971).

dev_ppp.txt
33:# NEED: these ioctl's are also defined for sock_pppox. We need multiple inheritance here (#322).

fs_ioctl_ext4.txt
23:# and/or if we mount tmpfs for sandbox=none (#971).
```
and `rg "github.*pull"` shows one:
```
dev_infiniband_rdma.txt
5:# https://github.com/google/syzkaller/pull/1053
```
So I think it will be fine to refer a PR by ID?